test/generators/html/gen_html/gen_html.ml

```diff
@@ -0,0 +1,92 @@
+open Sexplib0.Sexp
+
+type sexp = t = Atom of string | List of sexp list
+
+let read_lines ic =
+  let lines = ref [] in
+  try
+    while true do
+      lines := input_line ic :: !lines
+    done;
+    assert false
+  with End_of_file -> List.rev !lines
+
+let lines_of_command cmd args =
+  let inp = Unix.open_process_in (Filename.quote_command cmd args) in
+  let lines = read_lines inp in
+  let status =
+    match Unix.close_process_in inp with
+    | Unix.WEXITED n -> `Exit n
+    | _ -> `Signal
+  in
+  (lines, status)
+
+let expected_html_targets odocl =
+  lines_of_command "odoc" [ "html-targets"; "-o"; "html.gen"; odocl ]
+
+let tweak_html_target target =
+  String.(concat "." (split_on_char '/' (sub target 14 (length target - 14))))
```

Comment:
What is this constant "14" ? If you need a specific occurrence, I suggest you use pattern matching:
```ocaml
match split_on_char '/' target with
| _ :: _ :: rest -> concat ...
| _ -> assert false
```

Reply:
Done!